.env

```
@@ -361,4 +362,4 @@ VITE_DEBRIDGE_API_URL=https://dln.debridge.finance/v1.0
 VITE_USERBACK_TOKEN=A-3gHopRTd55QqxXGsJd0XLVVG3
 
 # agentic chat
-VITE_AGENTIC_SERVER_BASE_URL=https://api.agent.shapeshift.com
+VITE_AGENTIC_SERVER_BASE_URL=https://api.agent.shapeshift.com
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)